python/GafferUI/SpreadsheetUI/_CellPlugValueWidget.py

```diff
@@ -76,10 +77,21 @@ def __init__( self, plugOrPlugs, **kw ) :
 				self.__row.append( plugValueWidget )
 			else :
 				self.__row.append( GafferUI.Label( "Unable to multi-edit values of this type" ) )
```

Comment:
I know this isn't new, but "Unable to multi-edit" is beginning to sound a bit odd to me. Maybe "Unable to edit multiple plugs of this type"? Likewise, perhaps "Unable to edit plugs with mixed types" below?

Reply:
Reworded [here](https://github.com/GafferHQ/gaffer/pull/4001/commits/a6e0181e7e987766f59f90878d809ed1f3457a1b).